explainer.md

```diff
@@ -1,133 +1,156 @@
+The promise passed to `waitUntil` is a signal to the UA that the sync event is ongoing and that it should keep the SW alive if possible. Rejection of the event signals to the UA that the sync failed. Upon rejection the UA should reschedule (likely with a UA-determined backoff).
 
-## Handling Synchronization Events
+The UA may coalesce syncronizations to reduce the number of times the device, radio and browser need to wake up. The coalescing can be across origins, or even OS. Although the event timings are coalesced, you still get an event per pending sync registration.
 
-Synchronization happens from the Service Worker context via the new `sync` event. It is passed the id from the invoking request.
+## Periodic syncronization
 
-```js
-// sw.js
-self.onsync = function(event) {
+Opening a news or social media app to find content you hadn't seen before - without going to the network, is a user experience currently limited to native apps.
 
-  if (event.registration.id === "periodicSync") {
-    event.waitUntil(doAsyncStuffWithIndexedDBData());
-  } else {
-    // Delete unknown syncs (perhaps from older pages).
-    event.registration.unregister();
-  }
-};
+[The push API](https://w3c.github.io/push-api/) allows the server to dictate when the service worker should wake up and seek updates, but these are not sensitive to connection and charging state. Also, some sites update too frequently to warrant a push message per update (think Twitter, or a news site).
+
+Periodic syncs are simple to set up, don't require any server configuration, and allow the UA to optimize when they fire to be most-helpful and least-disruptive to the user. E.g. if the UA knows the user has a morning alarm set, it may run syncronizations shortly beforehand, giving the user quick and up-to-date information from their favourite sites.
+
+### The API
+
+**To request a periodic sync:**
+
+```js
+navigator.serviceWorker.ready(function(registration) {
+  registration.periodicSync.register({
+    tag: 'get-latest-news',         // default: ''
+    minPeriod: 12 * 60 * 60 * 1000, // default: 0
+    powerState: 'avoid-draining',   // default: 'auto'
+    networkState: 'avoid-cellular'  // default: 'online'
+  }).then(function(periodicSyncReg) {
+    // success
+  }, function() {
+    // failure
+  })
+});
 ```
-The `waitUntil` is a signal to the UA that the sync event is ongoing and that it should keep the SW alive if possible. Rejection of the event signals to the UA that the sync failed. Upon rejection the UA should reschedule (likely with a UA-determined backoff).
 
-## Removing Sync Events
-If the id is not registered the function will reject.
+* `tag`: This operates like a notification's tag. If you register a sync and an existing sync with the same tag is pending, it returns the existing registration. If further options are added to `.register`, the existing registration will be updated. **Note:** this tag is independent of one-off sync tags.
```

Comment:
s/this tag is independent of one-off-sync tags/one-off and periodic sync tags have separate namespaces/ ?